.cloudbuild/cloudbuild-test-b.yaml

```diff
@@ -19,7 +19,9 @@ substitutions:
 steps:
   # GraalVM B build
   - name: gcr.io/cloud-builders/docker
-    args: ["build", "-t", "gcr.io/cloud-devrel-public-resources/graalvm_b:${_JAVA_SHARED_CONFIG_VERSION}", "--file", "graalvm-b.Dockerfile", "."]
```

Comment:
This file is for testing, right? Then it's not needed.

Reply:
Yup this is only for testing, done.